gravity.sh

```diff
@@ -68,6 +68,17 @@ else
   exit 1
 fi
 
+# Source pihole-FTL from install script
+pihole_FTL="${piholeDir}/pihole-FTL.conf"
+if [[ -f "${pihole_FTL}" ]]; then
+  source "${pihole_FTL}"
+  if [[ -z "${BLOCKINGMODE}" ]] ; then
+    BLOCKINGMODE="Default (NULL)"
+  fi
+else
+    BLOCKINGMODE="Default (NULL)"
+fi
```

Comment:
This logic can be simplified:
```bash
if [[ -f "${pihole_FTL}" ]]; then
  source "${pihole_FTL}"
fi

if [[ -z "${BLOCKINGMODE}" ]] ; then
  BLOCKINGMODE="NULL"
fi
```

Also, just use `NULL` instead of `Default (NULL)` to avoid confusion.

Reply:
@Mcat12 , see the commit I pushed to this branch for